src/TableFunctions/TableFunctionRemote.cpp

```diff
@@ -37,7 +37,7 @@ void TableFunctionRemote::parseArguments(const ASTPtr & ast_function, const Cont
 
     ASTs & args = args_func.at(0)->children;
 
-    const size_t max_args = is_cluster_function ? 3 : 5;
```

Comment:
I guess it may be useful for `remote` too, like `insert into function remote('node{1,2}', db, table, user, pass, rand())` to distributed data across nodes.

Reply:
Currently, it already support insert into `remote`,  but the first argument of `remote` table function must be address.

Reply:
@ucasFL Indeed, but it does not support distributing data across nodes (i.e. when you have multiple addresses), right?

Reply:
Yes, currently it just can have one address.

Reply:
Yes, it is.

Reply:
@azat Now, it should support distributing data across nodes already.